docs/general-development/how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online.md

```diff
@@ -230,6 +230,9 @@ If you have applications or components, that are causing your people search requ
 1. Try consolidating the requests into a single query. For example, instead of making 10 simultaneous queries for `WorkEmail:user1@constoso.com`, `WorkEmail:user2@constoso.com`,..., `WorkEmail:user10@contoso.com`, try the single query, `WorkEmail:user1@constoso.com WorkEmail:user2@constoso.com ... WorkEmail:user10@contoso.com`.
 1. Consider using the [Microsoft Graph API](/graph/search-concept-person) if a high-request-volume scenario (in excess of 25 requests per second) is truly necessary.
 
+### When accessing OneDrive sites
+When a client attempts to access many OneDrive site collections that do not exist in excess, we may throttle requests from that client's IP address. The client will receive an HTTP 429 response when accessing any OneDrive site collection during the throttling period.

```

Comment:
> that do not exist in excess

Grammar nit: consider rewording to make it clear that it's the access attempts that are in excess, not the site collections:

> When a client makes excessive attempts to access many OneDrive site collections that do not exist, ..